### DISCUSSION TITLE
Sendin quick responses with doorbell

### DISCUSSION THREAD

**Discussion by ghost** [fuatakgun/eufy_security#179] [Ideas]
Hi there,

As requested on the HA community I will put this here as a request.

I would like to have the option to send quick responses (the custom ones in particular) through Home Assistant, based on certain events (like doorbell press). For example I thought it would be fun to trigger some custom recorded (novelty) quick responses on a doorbell press on certain occasions.

With your help I figured out it is possible to send out quick responses on the Eufy doorbell through Home Assistant with the eufy_security.send_message service. I added it to a script. (Below for reference to anyone who is interested)

```
alias: Doorbell Response 3
sequence:
  - service: eufy_security.send_message
    data:
      message: |-
        {
          "messageId": "message_id_do_not_add_value_here", 
          "command": "device.quick_response",
          "serialNumber": "your_serial_no",
          "voiceId": 3,
          "value": true
        }
mode: single
```

I have recorded three custom quick reponses, but they do not have the voiceID values 4, 5 and 6. So this is not a solution for the purpose I have for it. It would be fun to have this included in your custom integration!

**Comment by fuatakgun** [comment]
can you create a issue to request a service for device.quick_response?
